openstf/pipeline/create_component_forecast.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+import structlog
+
+import pandas as pd
+
+import openstf.postprocessing.postprocessing as postprocessing
+
+
+def create_components_forecast_pipeline(pj, input_data, weather_data, split_coefs):
+    """Pipeline for creating a component forecast
+
+    Args:
+        pj (dict): Prediction job
+        input_data (pd.DataFrame): Input data fore the components forecast, this usually is a regular forecast
+        weather_data (pd.DataFrame): Weather data with 'radiation' and 'windspeed_100m' columns
+        split_coefs (dict): coefieicnts for the splitting that are determined earlier
+
+    Returns:
+        pd.DataFrame with component forecasts
```

Comment:
How does the return look like, which columns will be added to the components forecast for example?

Reply:
(Future, all these explanation would be unnecessary once we have defined our data models of course)